doc-Monitoring_Alerts_and_Reporting/topics/To_create_a_Hostd_Log_Threshold_Alert.adoc

```diff
@@ -10,20 +10,16 @@ The following example shows steps to check the host's log for a failure to valid
 . Click  image:1847.png[] (*Configuration*),  image:1862.png[] (*Add a new Alert*).
 . In the *Info* area:
 +
-
-image:1981.png[]
+image:hostdLodAlert.png[]
```

Comment:
This image isn't showing up, but I think it's just due to a tiny typo: should be hostdLogAlert.png I think.